Graphviz/DOT.sublime-syntax

```diff
@@ -1,62 +1,391 @@
 %YAML 1.2
 ---
-# http://www.sublimetext.com/docs/3/syntax.html
+# http://www.sublimetext.com/docs/syntax.html
+# https://graphviz.org/doc/info/lang.html
+
 name: Graphviz (DOT)
 scope: source.dot
+version: 2
 
 file_extensions:
   - gv
   - dot
+  - DOT
 
 first_line_match: |-
   (?xi:
-    ^ \s* // .*? -\*- .*? \b(dot|graphviz)\b .*? -\*-  # editorconfig
+    {{leading_wspace}} // .*? -\*- .*? \b(dot|graphviz)\b .*? -\*-  # editorconfig
   )
 
+###[ CONTEXTS ]################################################################
+
 contexts:
+  prototype:
+    - include: comments
+
   main:
-    - include: expressions
+    - include: fullline-comments-number-sign
+    - include: declarations
+    - include: attribute-statements
+    - include: node-statements
+    - include: attribute-lists
+    - include: embedded-html
+    - include: literals
+    - include: operators
+
+###[ COMMENTS ]################################################################
+
+  # https://graphviz.org/doc/info/lang.html#comments-and-optional-formatting
+
+  comments:
+    - include: line-comments
+    - include: block-comments
+
+  line-comments:
+    # NOTE: number-sign line comments are not part of prototype
+
+    - match: /{2}
```

Comment:
Why not simply `//`? It's shorter and seems the more common pattern being in use by other syntax definitions in this repo.

```suggestion
    - match: //
```